$OpenBSD: patch-src_client_h,v 1.1 2012/10/12 13:31:49 dcoppa Exp $

Correct speling error.
(upstream git commit 6087b18a630eb32df682a7fd90d88408aa99c726)

--- src/client.h.orig	Fri Oct 12 15:15:18 2012
+++ src/client.h	Fri Oct 12 15:15:39 2012
@@ -51,7 +51,7 @@
 
 #define BUFSIZE 8192
 #define UDP_BUFSIZE 512
-#define INACTICITY_TIMEOUT 15 /* Seconds */
+#define INACTIVITY_TIMEOUT 15 /* Seconds */
 #define MAX_CODECS 10
 #define MAX_TOKENSIZE 64
 #define MAX_TOKENS 32
